LevelWorld/LevelView.cpp

```diff
@@ -204,7 +218,7 @@ void LevelView::MessageReceived(BMessage *message)
 {
 	switch ( message->what )
 	{
-		case B_SIMPLE_DATA :
```

Comment:
keep that in there to "fall thru" to the B_REFS_RECEIVED. otherwise drag&droping a level won't work anymore.